code/campaign/build.gradle.kts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2024 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+plugins {
+    id("aep-library")
+}
+
+val mavenCoreVersion: String by project
+val functionalTestIdentityVersion: String by project
+val functionalTestLifecycleVersion: String by project
+
+aepLibrary {
+    namespace = "com.adobe.marketing.mobile"
+    enableSpotless = true
+    enableCheckStyle = true
+
+    publishing {
+        gitRepoName = "aepsdk-campaign-android"
+        addCoreDependency(mavenCoreVersion)
+    }
+}
+
+dependencies {
+    implementation("com.adobe.marketing.mobile:core:$mavenCoreVersion-SNAPSHOT")
+
+    // testImplementation dependencies provided by aep-library:
+    // MOCKITO_CORE, MOCKITO_INLINE, JSON
+
+    // androidTestImplementation dependencies provided by aep-library:
+    androidTestImplementation("com.adobe.marketing.mobile:identity:$functionalTestIdentityVersion-SNAPSHOT")
+    androidTestImplementation("com.adobe.marketing.mobile:lifecycle:$functionalTestLifecycleVersion-SNAPSHOT")
+}
```

Comment:
How about firebase and fasterxml, do you still need to add them here?

Reply:
Nope, we don't need them in functional tests. Firebase is required for the test app. (We are not using Firebase API for retrieving push tokens in the functional test)

The tests run fine without without fasterxml.. So I think good to be removed